AfrikaBurn/BurnElementsViewController.swift

```diff
@@ -78,6 +99,27 @@ class BurnElementsViewController: UIViewController {
             self?.tableView.handleRealmChanges(changes)
         }
     }
+    
+    func updateSearchResults(for searchController: UISearchController){
+        let searchText = searchController.searchBar.text
+        if (searchText == nil || searchText! == "") {
```

Comment:
this could be easier done using a guard statement:

```swift
guard let searchText = searchText, searchText.isEmpty == false else {
    return
}
```